qa/tasks/radosgw_admin.py

```diff
@@ -27,7 +27,6 @@
 
 #import pdb
 
-import tasks.vstart_runner
```

Comment:
please comment these out instead of removing, so the method to run locally is at least documented:
```py
# uncomment vstart_runner lines to run locally against a vstart cluster
#import tasks.vstart_runner
```